toolchain/lower/file_context.cpp

```diff
@@ -922,16 +927,11 @@ auto FileContext::GetLocForDI(SemIR::InstId inst_id) -> Context::LocForDI {
       GetAbsoluteNodeId(sem_ir_, SemIR::LocId(inst_id)).back());
 }
 
-auto FileContext::BuildVtable(const SemIR::Vtable& vtable)
+auto FileContext::BuildVtable(const SemIR::Vtable& vtable,
+                              SemIR::SpecificId specific_id)
     -> llvm::GlobalVariable* {
   const auto& class_info = sem_ir().classes().Get(vtable.class_id);
 
-  // Vtables can't be generated for generics, only for their specifics - and
-  // must be done lazily based on the use of those specifics.
-  if (class_info.generic_id.has_value()) {
-    return nullptr;
-  }
-
   Mangler m(*this);
   std::string mangled_name = m.MangleVTable(class_info);
```

Comment:
Do we need to pass the `specific_id` in here, and include it / a fingerprint of it in the mangling?

Reply:
Oh, yeah - f147606609ddd9aa281084ea66b1ed67d8895fc0